drivers/sshj/src/test/java/org/jclouds/sshj/config/SshjSshClientModuleTest.java

```diff
@@ -42,4 +50,29 @@ public void testConfigureBindsClient() {
             .password("password").build());
       assert connection instanceof SshjSshClient;
    }
+
+   private class SshjConfigModule extends AbstractModule {
+      @Override
+      protected void configure() {
+         Config config = new DefaultConfig();
+         config.setVersion("42");
+         bind(Config.class).toInstance(config);
+         List<SshjSshClientModule.AuthMethodFactory> methods = new ArrayList<SshjSshClientModule.AuthMethodFactory>();
+         methods.add(null);
+         bind(new TypeLiteral<List<SshjSshClientModule.AuthMethodFactory>>() {
+         }).annotatedWith(Names.named("jclouds.ssh.sshj-auth-methods")).toInstance(methods);
+      }
+   }
+
+   public void testConfigurationBinding() {
+      Injector i = Guice.createInjector(new SshjSshClientModule(), new SLF4JLoggingModule(), new SshjConfigModule());
+      Config config = ((SshjSshClientModule.Factory)i.getInstance(SshClient.Factory.class)).config;
+      assert config.getVersion().equals("42");
```

Comment:
I'm checking that config injected is the same as config in module, by checking version set on line 58.